src/main/java/org/kairosdb/datastore/cassandra/CassandraConfiguration.java

```diff
@@ -48,6 +48,25 @@
 
 	public static final String CASSANDRA_INDEX_TAG_LIST = "kairosdb.datastore.cassandra.index_tag_list";
 
+	public static final String USE_NEW_SPLIT_INDEX = "kairosdb.datastore.cassandra.new-split-index-start-time-ms";
```

Comment:
the values are confused. It must be `NEW_SPLIT_INDEX_START_TIME_MS` here instead of `USE_NEW_SPLIT_INDEX`. And vice versa on the next line.

Reply:
<facepalm/>

Reply:
\</facepalm\>